omod/src/main/webapp/addLabTestAttributeType.jsp

```diff
@@ -979,15 +979,9 @@ legend.scheduler-border {
 	}
 	
 	
-	function checkConceptExistent(concepId){
-		console.log("Concept ID : "+concepId);
-		console.log("integer : "+isInt(concepId));
-		if(isInt(concepId)){
-		return	getCheckExist(concepId);
-		}
-		else {
-			return false;
-		}
+	function checkConceptExists(conceptUuid){
```

Comment:
Rename to checkObjectExists(objectClass, objectUuid).
What about datatypes other than concept? All allowed datatypes should be handled.

Reply:
Other datatype don't need datatype config field.